project/BRANCHES-AND-TAGS.md

```diff
@@ -1,32 +1,39 @@
 
-The `docs` branch supports documentation updates between product releases. This branch allow us to
-decouple documentation releases from product releases.
+Currently (and previously) maintained release branches are documented in the table below:
 
-## `release` branch
+| Branch Name                                    | Sponsoring Maintainer(s)                       | Contribution Status   | Expected End of Maintenance  | Known Distributors                                                   |
+|------------------------------------------------|------------------------------------------------|-----------------------|------------------------------|----------------------------------------------------------------------|
+| master (development branch)                    | The Moby Project [MAINTAINERS](../MAINTAINERS) | N/A                   | -                            | N/A                                                                  |
+| 24.0                                           | @thaJeztah, @rumpl, @neersighted               | Maintained            | TBD                          | [Docker, Inc.](https://docker.com)                                   |
+| 23.0                                           | @corhere, @cpuguy83                            | Maintained            | TBD                          | [Mirantis](https://mirantis.com), [Microsoft](https://microsoft.com) |
+| 20.10                                          | @corhere, @cpuguy83                            | Maintained (security) | December 2023                | [Mirantis](https://mirantis.com), [Microsoft](https://microsoft.com) |
 
-The `release` branch contains the last _released_ version of the code for the project.
+> Note: The Moby Project provides source code releases. Binary distributions are available from multiple contributing parties, and known distributions can be discovered in [PACKAGERS.md](PACKAGERS.md).
 
-The `release` branch is only updated at each public release of the project. The mechanism for this
-is that the release is materialized by a pull request against the `release` branch which lives for
-the duration of the code freeze period. When this pull request is merged, the `release` branch gets
-updated, and its new state is tagged accordingly.
+## Contribution Status
+
+The contribution status of a branch is meant to set contributor expectations for acceptance of changes into a branch, as well as document what level of contribution or maintenance the sponsoring maintainers expect to perform. This status is informational and not binding.
+
+- **Maintained** - actively developed by project maintainers; accepting contributions and backports; in-scope for security advisories 
+- **Maintained (security)** - no longer actively developed; may accept contributions and backports for critical security issues; in-scope for security advisories
+- **Unmaintained** - no longer actively developed; not accepting contributions; out-of-scope for security advisories
```

Comment:
containerd [describes](https://containerd.io/releases/#support-horizon) these as "Active", "Extended", and "End of Life" in case you wanted to keep terminology consistent across projects.